content/shared/_channel_enumeration.textile

```diff
@@ -0,0 +1,22 @@
+The REST API now includes the endpoint:
```

Comment:
Not now, this is living documentation.  The REST API includes an endpoint to enumerate active channels within your app.